pmoves/chit/__init__.py

```
@@ -303,45 +332,62 @@ def sync_common_credentials(
-            "MINIO_ACCESS_KEY": "minioadmin",
-            "MINIO_SECRET_KEY": "minioadmin",
-            "MINIO_ROOT_USER": "minioadmin",
-            "MINIO_ROOT_PASSWORD": "minioadmin",
-            "MINIO_USER": "minioadmin",
-            "MINIO_PASSWORD": "minioadmin",
-            "NEO4J_AUTH": "neo4j/changeme",
-            "NEO4J_PASSWORD": "changeme",
-            "PGRST_DB_URI": "postgres://pmoves:changeme@postgres:5432/pmoves",
-        }
+        # Try os.environ first (set by CI, Docker Compose, or operator)
+        common_creds = {k: os.environ.get(k, "") for k in _COMMON_CRED_KEYS}
```

Ruff (0.15.10)

[error] Undefined name `os`

(F821)